### ISSUE TITLE
Results for iLIDS-VID

### ISSUE THREAD

**Issue by TragedyN** [deropty/PiT#3]
Me again hh , I used your given iLIDS-VID weight(https://pan.baidu.com/s/10LosWwUMktTiWvbHEP1Tjw) to do the inference task for iLIDS-VID dataset, I found the results are lower than I expected.  Maybe there is something wrong with my conda env? 

Results:
2022-04-28 10:11:14,088 pit INFO: rank_1:0.7866666913032532, rank_5 0.95333331823349 : trial : 9
2022-04-28 10:11:14,088 pit INFO: sum_rank_1:87.8%, sum_rank_5 97.5%

My env:
python                    3.6.13
torch                     1.10.1 
torchvision               0.11.2
cuda                        10.2

**Comment by TragedyN** [comment]
I changed my torch version to v1.8.1, results are still the same.

**Comment by deropty** [comment]
> I changed my torch version to v1.8.1, results are still the same.

I guess that you have found there are some bugs in test.py. Thus, I recommended doing the training & testing job by using the train.py file. It is beyond my thought that someone would directly DO the inference task. All right, you win! 

I will correct these bugs within 2 days, so stay tuned！

**Comment by deropty** [comment]
> I changed my torch version to v1.8.1, results are still the same.

The train.py has been updated. You can download the latest source codes in the OpenI.
For training and testing processes, the parameter `TEST.WEIGHT` in yml file should be NULL.
For testing only, the parameter `TEST.WEIGHT` should be the directory of the model.

I have tested our codes with PyTorch 1.6.0 and 1.8.1. The results are in [train_log.zip](https://github.com/deropty/PiT/files/8581752/train_log.zip), which are the same as the results in our paper. You can compare these results with yours.
i.e., 
2022-04-28 13:39:32,211 pit INFO: CMC curve, Rank-1  :92.067%
2022-04-28 13:39:32,211 pit INFO: CMC curve, Rank-5  :98.933%
2022-04-28 13:39:32,211 pit INFO: CMC curve, Rank-10 :99.800%
2022-04-28 13:39:32,211 pit INFO: CMC curve, Rank-20 :100.000%

Our env: python 3.6.13, torch 1.6.0 or 1.8.1, cuda 10.1.
By the way, the conda env may not affect the results.

**Comment by TragedyN** [comment]
Thx!!! I will try it later.

**Comment by TragedyN** [comment]
![image](https://user-images.githubusercontent.com/33872545/169730478-2014869c-db9c-4849-8a3a-afbb7f248ec4.png)
Me again! Still iLIDS-VID, in my training process, I found that in the 8th split, just like the pic, the results were so wrong, and in the other splits, the rank-1 was about 88%-90%, I don't know what happened.

**Comment by deropty** [comment]
> ![image](https://user-images.githubusercontent.com/33872545/169730478-2014869c-db9c-4849-8a3a-afbb7f248ec4.png) Me again! Still iLIDS-VID, in my training process, I found that in the 8th split, just like the pic, the results were so wrong, and in the other splits, the rank-1 was about 88%-90%, I don't know what happened.

It is strange. I have never encountered this problem. The 'nan' error may come from gradient explosion. If you have multiple types of GPUs, do experiments on another one. Maybe it works, I guess.
